kernel/libc/stdio.c

```diff
@@ -65,7 +65,7 @@ void ftoa(double num, char *buffer, int precision)
 	}
 
 	buffer[len + i] = '\0';
-	reverse(buffer, len + i);
+	// reverse(buffer, len + i);
```

Comment:
You can delete the reverse call instead of commenting no problem at all

Reply:
Should be done now.